src/js/Site.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import ProductList from './components/productList/ProductList.jsx'
+import CurrentFilterInfo from './components/currentFilterInfo/CurrentFilterInfo.jsx'
+import FilterList from './components/filterList/FilterList.jsx'
+import ProductListFooter from './components/productList/ProductListFooter.jsx'
+import ProductListViewSettings from './components/productList/ProductListViewSettings.jsx'
+import StarRating from './components/general/StarRating.jsx'
+
+const productData=[
+    {name: "Automobil", size: '90x55mm'},
+    {name: "Killum", size: '90x55mm'},
+    {name: "Severamagenition", size: '90x55mm'},
+    {name: "Abrahamoska", size: '90x55mm'},
+    {name: "Fifth", size: '90x55mm'},
+    {name: "Runner", size: '90x55mm'},
+];
+
+const filterListData = {
+    ["OrientationFilter"]: {
+        type: 'orientation'
+    },
+    ["Size"]: {
+        type: 'checkbox',
+        filterValues: [
+            {name: '90x50mm', filteredElementsAmount: 112},
+            {name: '85x55mm', filteredElementsAmount: 67}
+        ]
+    },
+    ["Customer Rating"]: {
```

Comment:
zrób bez tej spacji. W OrientationFilter tak zrobiłaś i też tak może być
imho jak nie robisz spacji w kluczu to możesz od razu go podać z pominięciem [] czyli nie ["Size"] a od razu Size: